src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcEndpointDataSource.cs

```diff
@@ -383,7 +383,7 @@ private IChangeToken GetCompositeChangeToken()
             return new CompositeChangeToken(changeTokens);
         }
 
-        public override IChangeToken ChangeToken => GetCompositeChangeToken();
```

Comment:
Just double checking, you make `ChangeToken` virtual on the base class right? So we don't break when we remove our o/l?

Reply:
Ya it is virtual, otherwise it wouldn't compile here.